actor/src/deadlock.rs

```diff
@@ -1,6 +1,10 @@
 //! Module which is used for deadlock detection of actors
 
-#![allow(clippy::expect_used, clippy::panic)]
+#![allow(
+    clippy::expect_used,
+    clippy::panic,
+    clippy::declare_interior_mutable_const
```

Comment:
may I ask what activates this lint?

Reply:
Essentially, anyhting that defines a const with interior mutability. Normally I'd say fix it. But in this particular case it comes from code generated by a macro.